### ISSUE TITLE
Add blame page to file view

### ISSUE THREAD

**Issue by hhenkel** [gogits/gogs#2606]
Github gives us to look at a blame page for a file:
![github](https://cloud.githubusercontent.com/assets/2813782/13007516/2d5f6ae6-d191-11e5-9aaa-f8c30d0cf55d.png)
Resulting in something like https://github.com/gogits/gogs/blame/master/CONTRIBUTING.md

It would be nice to have this feature in Gogs as well.

**Comment by zhukovra** [comment]
Hi, @unknwon. I want to try implement that. Please, review needed changes in https://github.com/gogs/git-module/pull/62

**Comment by Elikill58** [comment]
Even if the PR have been accepted, on the main website I can't find the button. At which step is this feature ?

**Comment by unknwon** [comment]
> Even if the PR have been accepted, on the main website I can't find the button. At which step is this feature ?

I don't think this is supported at all, from what I can tell only the changes to `gogs/git-module` is merged, nothing changed in the Gogs main application codebase 🤔
